### DISCUSSION TITLE
Fill and stroke classes

### DISCUSSION THREAD

**Discussion by garethworld** [tailwindlabs/tailwindcss#4550] [Ideas]
Now we have JIT,  it would be great to have classes for fill and stroke that use the same colours values as text, bg etc.. The current-fill, current-stroke classes are quite limiting .

**Comment by VicGUTT** [comment]
Hey !
Does [customizing the fill colors](https://tailwindcss.com/docs/fill#customizing) & [customizing the stroke colors](https://tailwindcss.com/docs/stroke#customizing) not work well for you ?

To have your full color set for fill & stroke I believe you could simply add this to your `tailwind.config.js` file:

```js
{
    theme: {
        fill: theme => ({ ...theme('colors') }),
        stroke: theme => ({ ...theme('colors') }),
    },
}
```

**Comment by garethworld** [comment]
thanks! that is exactly what I needed, but I don't see a reason this could not be the default now that file size is not an issue.

**Reply by VicGUTT** [reply]
Well, as long as JIT mode is not the default (as far as I know it will be in v3 by the end of this year) file size will still be an issue. Plus there's also different builds provided by Tailwind, think the CDN version etc... . So a conscious choice still needs to be made.

Hope it helps ✌🏾

**Comment by MichaelAllenWarner** [comment]
Bumping this now that v3 alpha is out. With JIT "all the time," it's worth considering making the `fill` and `stroke` utilities use `colors` by default.
